tools/crmadmin.c

```diff
@@ -16,32 +16,18 @@
 #include <glib.h>               // gboolean, GMainLoop, etc.
 #include <libxml/tree.h>        // xmlNode
 
-#include <crm/crm.h>
-#include <crm/cib.h>
-#include <crm/msg_xml.h>
+#include <pacemaker-internal.h>
+#include <pcmki/pcmki_cluster_queries.h>
+#include <pcmki/pcmki_output.h>
```

Comment:
pacemaker-internal.h includes pcmki/*, so you don't need to include those separately